make/scripts/fixpath.sh

```diff
@@ -145,10 +145,13 @@ function import_path() {
   fi
 
   if [[ "$path" != "" ]]; then
+    # Store current unix path
+    unixpath="$path"
     # Now turn it into a windows path
     winpath="$($PATHTOOL -w "$path" 2>/dev/null)"
     # If it fails, try again with an added .exe (needed on WSL)
     if [[ $? -ne 0 ]]; then
+      unixpath="$unixpath.exe"
       winpath="$($PATHTOOL -w "$path.exe" 2>/dev/null)"
```

Comment:
I think it would be a little cleaner/clearer to use unixpath on line 155.

Reply:
We will check the result from `$PATHTOOL` at L154, so I think corrent location is fine.